src/compiler/crystal/config.cr

```diff
@@ -15,13 +15,17 @@ module Crystal
     end
 
     def self.description
-      formatted_sha = "[#{build_commit}] " if build_commit
-      <<-DOC
-        Crystal #{version} #{formatted_sha}(#{date})
+      String.build do |io|
+        io << "Crystal " << version
+        io << " [" << build_commit << "]" if build_commit
+        io << " (" << date << ")" unless date.empty?
 
-        LLVM: #{llvm_version}
-        Default target: #{self.host_target}
-        DOC
+        io << "\n\nLLVM: " << llvm_version
+        io << "\nDefault target: " << host_target
+
+        io << "\n\nThe compiler was not built in release mode." unless release_mode?
```

Comment:
In light of https://github.com/crystal-lang/crystal/pull/13254#issuecomment-1495950950 I would suggest to put this not in the last line. I think it also makes more sense semantically as a second line, after the compiler build info, and before the LLVM version.

Reply:
What if someone else tries to get the host target from `Crystal::DESCRIPTION.lines[3]` instead? You cannot avoid a breakage either way so I wouldn't worry about which line the new text shows up on

Reply:
Yeah, it's not a strong improvement (although I would argue that `lines[-1]` is much more obvious than `lines[3]`).
But I also think it makes more sense semantically after the build info.